### ISSUE TITLE
CMake unconditionally installs libint2.pc into lib/pkgconfig

### ISSUE THREAD

**Issue by mbanck** [evaleev/libint#230]
The cmake build systems decides to override the standard CMAKE_INSTALL_LIBDIR with LIBINT2_INSTALL_LIBDIR (why?) for the libraries, but it uses a hardcoded `lib/pkgconfig` directory for `libint2.pc` (but see #225 - the generated `libint2.pc` is broken anyway):

```
# install cmake-processed files
install(FILES ${PROJECT_BINARY_DIR}/libint2.pc
        DESTINATION lib/pkgconfig)
```

This leads to `libint2.pc` not ending up in the same place as the library (they go into `/usr/lib/<gnu-triplet>/` on most distributions in order to support multiarch).

**Comment by loriab** [comment]
I think this is addressed in #233 .
* `CMAKE_INSTALL_LIBDIR` from GNUInstallDirs is left alone and can be overridden by user (no more `LIBINT2_INSTALL_LIBDIR`)
* paths to the basis set directory and cmake files directory are initialized from GNUInstallDirs but also have `LIBINT2_` handles. https://github.com/evaleev/libint/pull/233/files#diff-8e69ff48dd421c27038dafc327ded859e9be46b2e1aecec070d8b263a7b03d02R79-R92
* pkgconfig file location is hard-coded to `CMAKE_INSTALL_LIBDIR/pkgconfig/` https://github.com/evaleev/libint/pull/233/files#diff-8e69ff48dd421c27038dafc327ded859e9be46b2e1aecec070d8b263a7b03d02R645 . If you want a variable so that can be adjusted, let me know.

**Comment by mbanck** [comment]
Sounds good on first glance; I'm sorry, but I don't have the bandwidth right now to check out that #233 PR, I'll try to give it a spin with respect to distribution packaging ASAP (unless @susilehtola did already?)

**Comment by susilehtola** [comment]
I haven't updated libint2 in Fedora since Aug 2019 when 2.6.0 was released...

**Comment by susilehtola** [comment]
I've been waiting for the issues with Psi4 (including CMake in libint2) to be addressed

**Comment by loriab** [comment]
> Sounds good on first glance; I'm sorry, but I don't have the bandwidth right now to check out that #233 PR

Sure, no problem. #233 isn't ready for trial yet, and it needs EFV approval, too. I just wanted to record that this issue should be fixed up.

@susilehtola, yes, I'm working on the gmp/mpfr/eigen detection.

**Comment by loriab** [comment]
This should be minimally addressed in #271 . The library is still going to LIBINT2_INSTALL_LIBDIR because I'm not making unnecessary changes, but the pc file goes to CMAKE_INSTALL_LIBDIR/pkgconfig. Thus locations are adjustable.
